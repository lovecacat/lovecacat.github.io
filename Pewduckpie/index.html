<!DOCTYPE html>
<html lang="en-US">
  <head>
    <meta charset="UTF-8">
    <title>PewDuckPie</title>
    
    
    
    
        <link rel="stylesheet" href="css/style.css">

     <link rel="icon" a href="https://developers.google.com/games/services/images/branding/ic_play_games_badge_green.png" type="image/x-icon"/>
    
    
  </head>

<body>
	<title>The PewDuckPie Jump</title>



<div><audio src="name.mp3" autoplay="autoplay">
  <embed src="name.mp3" autostart="true" loop="true" hidden="true">
</audio></div>


	<div class="container">
		<canvas id="canvas">
			Aww, your browser doesn't support HTML5!
		</canvas>

		<div id="mainMenu">
			<h2>The PewDuckPie Jump</h2>

			<p class="info">
				use
				<span class="key left">←</span>
				<span class="key right">→</span>
				to move and space to (re) start...
			</p>
			<a class="button" href="javascript:init()">Play</a>
		</div>
		
		<div id="gameOverMenu">
			<h1>Oh U Die=]Poor</h1>
			<h3 id="go_score">you scored 0 points</h3>

			<a class="button" href="javascript:reset()">Restart</a>
			<a id="tweetBtn" target="_blank" class="button tweet" href="#">Tweet score</a>

			<a id="fbBtn" target="_blank" class="button fb" href="#">Post on FB</a>
		</div>
		
		<!-- Preloading image ;) -->
		<img id="sprite" src="http://s23.postimg.org/xewmynzmx/pewduckpie.png"/>

		<div id="scoreBoard">
			<p id="score">0</p>
		</div>

	</div>

   <script src="js/index.js"></script>
</body>
</html>
