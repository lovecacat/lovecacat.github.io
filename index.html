<!DOCTYPE HTML>
<!--
	Identity by HTML5 UP
	html5up.net | @n33co
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Duck is coming</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<!--[if lte IE 8]><script src="assets/js/html5shiv.js"></script><![endif]-->
		<link rel="stylesheet" href="assets/css/main.css" />
		<!--[if lte IE 9]><link rel="stylesheet" href="assets/css/ie9.css" /><![endif]-->
		<!--[if lte IE 8]><link rel="stylesheet" href="assets/css/ie8.css" /><![endif]-->
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-loading">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<section id="main">
						<header>
							<span class="avatar"><img src="https://scontent-hkg3-1.cdninstagram.com/t51.2885-19/s150x150/12277548_1705352009680737_1634258385_a.jpg" alt="width=100%"/></span>
							<h1>Pewdiepie</h1>
							<p>Don't be a salad</p>
						</header>
						<!--
						<hr />
						<h2>Extra Stuff!</h2>
						<form method="post" action="#">
							<div class="field">
								<input type="text" name="name" id="name" placeholder="Name" />
							</div>
							<div class="field">
								<input type="email" name="email" id="email" placeholder="Email" />
							</div>
							<div class="field">
								<div class="select-wrapper">
									<select name="department" id="department">
										<option value="">Department</option>
										<option value="sales">Sales</option>
										<option value="tech">Tech Support</option>
										<option value="null">/dev/null</option>
									</select>
								</div>
							</div>
							<div class="field">
								<textarea name="message" id="message" placeholder="Message" rows="4"></textarea>
							</div>
							<div class="field">
								<input type="checkbox" id="human" name="human" /><label for="human">I'm a human</label>
							</div>
							<div class="field">
								<label>But are you a robot?</label>
								<input type="radio" id="robot_yes" name="robot" /><label for="robot_yes">Yes</label>
								<input type="radio" id="robot_no" name="robot" /><label for="robot_no">No</label>
							</div>
							<ul class="actions">
								<li><a href="#" class="button">Get Started</a></li>
							</ul>
						</form>
						<hr />
						-->
						<footer>
							<ul class="icons">
								<li><a href="https://twitter.com/pewdiepie" target="_blank" class="fa-twitter">Twitter</a></li>
								<li><a href="https://www.instagram.com/pewdiepie/" target="_blank" class="fa-instagram">Instagram</a></li>
								<li><a href="https://www.facebook.com/PewDiePie/" target="_blank" class="fa-facebook">Facebook</a></li>
							</ul>
						</footer>
						<p>
						<br>
						<img src="http://vignette2.wikia.nocookie.net/the-house-of-anubis/images/a/af/Pewdiepie-Felix-inspiration.gif/revision/latest?cb=20140213184325">
						<hr>
						<br>
						<i>Don't be a salad,be the best goddamn broccoli you could ever be!-Pewdiepie</i>
						<br>
						<div class="intro"><h3><b>Felix Arvid Ulf Kjellberg,Everyone known as PewDiePie.He is a Swedish and a Youtuber.He loves all his bros more than anyone did.</b></h3></div>
						<br>
						<h2>*Brofist*</h2>
						<br>
						Actually,pewds has two cute dogs-Edgar&Maya. <img src="https://s-media-cache-ak0.pinimg.com/236x/ce/41/7b/ce417be24693b29bf8bbcc87dd88e6b8.jpg">
						<br>
						<br>
						<br>
						<h4>PewDiePie Shop</h4><a href="http://shop.pewdiepie.com/?utm_source=pewdiepie&utm_medium=yt_description&utm_campaign=shop"><img src="http://www.pewdiepie.com/media/img/site-logo-sm.png"></a>
							<br>Thanks for all your support bros, rating the video and leaving a comment is always appreciated! 
							<br>
							<br><b><h4>HOW TO BE A SALAD! - (Fridays With PewDiePie - Part 60)</h4></b>
														<br><iframe width="560" height="315" src="https://www.youtube.com/embed/n1WQ1B21xx0" frameborder="0" allowfullscreen></iframe>">
														<br><br><br><h4><b><a href="https://www.youtube.com/channel/UCqVDpXKLmKeBU_yyt_QkItQ">YoutubeRed</a>:Scare Pewdiepie(Preview)</b></h4>
														<br><iframe width="560" height="315" src="https://www.youtube.com/embed/0-fyoq4454U?list=PLjq6DwYksrzy3apz-yMjwSOJUKUqeYaaj" frameborder="0" allowfullscreen></iframe>
														<br>It is <u>HKD16</u> now.
						</p>
						<hr>
						<p>
						<!-- begin wwww.htmlcommentbox.com -->
 <div id="HCB_comment_box"><a href="http://www.htmlcommentbox.com">Comment Form</a> is loading comments...</div>
 <link rel="stylesheet" type="text/css" href="//www.htmlcommentbox.com/static/skins/bootstrap/twitter-bootstrap.css?v=0" />
 <script type="text/javascript" id="hcb"> /*<!--*/ if(!window.hcb_user){hcb_user={};} (function(){var s=document.createElement("script"), l=hcb_user.PAGE || (""+window.location).replace(/'/g,"%27"), h="//www.htmlcommentbox.com";s.setAttribute("type","text/javascript");s.setAttribute("src", h+"/jread?page="+encodeURIComponent(l).replace("+","%2B")+"&opts=16862&num=10&ts=1459323285574");if (typeof s!="undefined") document.getElementsByTagName("head")[0].appendChild(s);})(); /*-->*/ </script>
<!-- end www.htmlcommentbox.com -->
</p>
<p><iframe width="560" height="315" src="https://www.youtube.com/embed/BmxCR4_abd4?rel=0&autoplay=1" frameborder="0" allowfullscreen></iframe></p>
					</section>
<p><h1><b>stay awesome you bros~</b></h1></p>
<hr>

				<!-- Footer -->
					<footer id="footer">
						<ul class="copyright">
							<li>&copy; Pewdiepie</li>
							<li>Design: <a href="https://www.instagram.com/cwing__tung__614/">K.C.</a><i class="fa fa-user fa-2x"></i></li>
						</ul>
					</footer>

			</div>

		<!-- Scripts -->
			<!--[if lte IE 8]><script src="assets/js/respond.min.js"></script><![endif]-->
			<script>
				if ('addEventListener' in window) {
					window.addEventListener('load', function() { document.body.className = document.body.className.replace(/\bis-loading\b/, ''); });
					document.body.className += (navigator.userAgent.match(/(MSIE|rv:11\.0)/) ? ' is-ie' : '');
				}
			</script>

	</body>
</html>