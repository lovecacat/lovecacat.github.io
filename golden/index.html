<!DOCTYPE html>
<html >
  <head>
    <meta charset="UTF-8">
    <title>Memory Game</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
    <link rel="stylesheet" href="css/reset.css">

    <link rel='stylesheet prefetch' href='http://fonts.googleapis.com/css?family=Anton'>

        <link rel="stylesheet" href="css/style.css">

    <link rel="icon" a href="https://developers.google.com/games/services/images/branding/ic_play_games_badge_green.png" type="image/x-icon"/>
    
    
  </head>

  <body>

    <div class="wrap">
<div class="game"></div>
	
	<div class="modal-overlay">
		<div class="modal">
		
			<h2 class="winner"><img src="http://pic.pimg.tw/hahapoint/1307887762-fc3c9ab45ac11545f7dc9705e5562509.jpg" alt="100%" /><embed src="DEADPOOL.mp3" autostart="true" loop="true" hidden="true"></h2>
			<button class="restart">Play Again?(Don't)</button>
			<p class="message">Developed on <a href="http://codepen.io">CodePen</a> by <a href="http://codepen.io/lovecacat">Eggplant</a></p>
			<p class="share-text">Share it?</p>
			<ul class="social">
				<li><a target="_blank" class="twitter" href="https://twitter.com/pewdiepie"><span class="brandico-twitter-bird"></span></a></li>
				<li><a target="_blank" class="facebook" href="https://www.facebook.com/PewDiePie/"><span class="brandico-facebook"></span></a></li>
			</ul>
		</div>
	</div>
  <footer>
		<p class="disclaimer">All logos are property of their respective owners, No Copyright infringement intended.</p>
	</footer>
  </div><!-- End Wrap -->
<body>
<embed src="DEADPOOL.mp3" autostart="true" loop="true" hidden="true">
</body>
    <script src='http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js'></script>

        <script src="js/index.js"></script>

    
    
    
  </body>
</html>
